openfl/component/aggregation_functions/median.py

```diff
@@ -11,7 +13,7 @@
 class Median(AggregationFunction):
     """Median aggregation."""
 
-    def call(self, local_tensors, *_) -> np.ndarray:
+    def call(self, local_tensors: List, *_) -> np.ndarray:
```

Comment:
Should we add this or not?
```suggestion
    def call(self, local_tensors: List, *_: Any) -> np.ndarray:
```